src/sage/rings/tate_algebra_ideal.pyx

```diff
@@ -623,7 +623,7 @@ cdef TateAlgebraElement regular_reduce(sgb, TateAlgebraTerm s, TateAlgebraElemen
 
     TESTS::
 
-        sage: cython(                                                                   # needs sage.misc.cython
+        sage: cython(                                                                   # long time 6.5s, needs sage.misc.cython
```

Comment:
```suggestion
        sage: cython(                                                                   # long time (6.5s), needs sage.misc.cython
```
parentheses please -- throughout the changes

Reply:
Sorry, I have fixed *every* that I can find in the source code :)